keyboards/q4z/readme.md

```diff
@@ -0,0 +1,10 @@
+# Q4Z
```

Comment:
This file should follow the [template](https://github.com/qmk/qmk_firmware/blob/master/quantum/template/avr/readme.md).